app/models/submission_viewing_event.rb

```diff
@@ -0,0 +1,2 @@
+class SubmissionViewingEvent < ActiveRecord::Base
```

Comment:
model file auto generated with 'rails generate model' - not used currently but may be helpful for future work with data table